_sass/image.scss

```diff
@@ -0,0 +1,28 @@
+.image {
+  position: relative;
+  text-align: center;
+  a {
+    text-decoration: none;
+    i {
+      position: absolute;
+      top: 50%;
```

Comment:
@Conor12345 Fell free to play around with this parameters to align it the way you want.
`top:` and `bottom:`